config/flask_config.py

```diff
@@ -67,14 +101,20 @@
 ############################################################################
 
 # Base directory
+config['BASEDIR'] = path.abspath(path.join(path.dirname(__file__), pardir))
 BASEDIR = path.abspath(path.join(path.dirname(__file__), pardir))
 
+config['RELATIVE_UPLOAD_FOLDER'] = 'app/static/img/uploaded/'
 RELATIVE_UPLOAD_FOLDER = 'app/static/img/uploaded/'
+config['UPLOAD_FOLDER'] = path.join(config['BASEDIR'], config['RELATIVE_UPLOAD_FOLDER'])
 UPLOAD_FOLDER = path.join(BASEDIR, RELATIVE_UPLOAD_FOLDER)
+config['RELATIVE_DELETE_FOLDER'] = 'app/static/img/uploaded/deleted/'
 RELATIVE_DELETE_FOLDER = 'app/static/img/uploaded/deleted/'
+config['DELETE_FOLDER'] = path.join(config['BASEDIR'], config['RELATIVE_DELETE_FOLDER'])
 DELETE_FOLDER = path.join(BASEDIR, RELATIVE_DELETE_FOLDER)
```

Comment:
Let's not have any of these variables, and just add them onto the `config` object.  We can import `config` whenever we want if we need these variables in other modules.  **Make sure to grep to make sure removing them doesn't break anything.  I would look for `from app.config`**